classes/migration/upgrade/v3_4_0/UpgradeMigration.inc.php

```diff
@@ -16,9 +16,15 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Schema;
+use APP\core\Application;
+use PKP\submissionFile\SubmissionFile;
+use APP\core\Services;
 
 class UpgradeMigration extends \PKP\migration\Migration
```

Comment:
@defstat, could you move this migration to its own new class (named after the issue number, as with most of the other examples in `lib/pkp/classes/migration/upgrade/v3_4_0`)? Shorter upgrade scripts, with easy tracking back to the issue they fix, will help keep the upgrade system maintainable (and debugging easier).

Reply:
Sure. Fixed.